llvm/include/llvm/IR/RuntimeLibcalls.td

```diff
@@ -1151,7 +1187,10 @@ def isAArch64_ILP64  : RuntimeLibcallPredicate<"TT.isAArch64(64)">;
 
 def AArch64SystemLibrary : SystemRuntimeLibrary<
   isAArch64_ExceptArm64EC,
-  (add DefaultRuntimeLibcallImpls,
+  (add Win64DefaultLibcallImpls,
+       LibcallImpls<(add __powisf2, __powidf2), isNotOSMSVCRT>,
+       LibmHasFrexpF32, LibmHasLdexpF32,
+       LibmHasFrexpF128, LibmHasLdexpF128,
        AArch64LibcallImpls,
```

Comment:
Might be missing something here, but it doesn't look like `AArch64SystemLibrary` is only for Windows. Why did it change to `Win64DefaultLibcallImpls` rather than `DefaultRuntimeLibcallImpls`?

Reply:
Because Win64DefaultLibcallImpls is the subset supported by windows. Everything else is additive for different not windows